backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtMusic.java

```diff
@@ -117,6 +117,18 @@ public float getPosition () {
 		return sound.getPosition() / 1000f;
 	}
 
+	@Override
+	public void setPlaybackRate(float playbackRate) {
+		if (playbackRate >= 0.5f && playbackRate <= 4.0f) {
```

Comment:
The `Music` interface says that `playbackRate`s should be greater equal than 1, so why allow lower values here?

Reply:
That's a misunderstanding of my doc string (see the other review)